pkg/rtc/transport.go

```diff
@@ -473,8 +473,6 @@ func (t *PCTransport) onICEConnectionStateChange(state webrtc.ICEConnectionState
 		} else {
 			t.params.Logger.Infow("selected ICE candidate pair", "pair", pair)
 		}
-	case webrtc.ICEConnectionStateFailed:
-		t.handleConnectionFailed()
```

Comment:
I guess this was removed because ICE failed also results in peer connection failed and we catch it there?

Reply:
yeah it always go there, so remove the ice failed handle here to avoid fire short connection twice